fixit/cli/run_rules.py

```diff
@@ -163,9 +178,6 @@ def main(raw_args: Sequence[str]) -> int:
             + f"{time.time() - start_time :.2f} seconds."
         )
 
-    # Return with an exit code of 1 if there are any violations found.
-    return int(bool(formatted_reports))
```

Comment:
`run_rule` command can be run in builds and the return code will be used to fail the builds.
Can you keep this behavior unchanged?
It should also be added to `main.py`. Each _main can return an int as an argument for `sys.exit`.